integration-tests/vm.install

```diff
@@ -14,7 +14,7 @@ tar xzf ../subscription-manager-build-extra.tar.gz
 python3 ./setup.py build
 python3 ./setup.py install --prefix /usr
 
-mv /usr/bin/{rhsm-facts-service,rhsm-service,rhsmcertd-worker} /usr/libexec
```

Comment:
Unfortunately this won't work: sadly the current buildsystem has a mixture of `setup.py` & `Makefile` bits, so only using `setup.py` does not install everything. (In case you wonder: yes, we know about this, and we would like to fix this somewhen in the future.)

A possible alternative to simplify this would be to switch from `setup.py` to `Makefile` for build and installation. It could be a good idea for this script; I'd do it as separate work/PR though, mostly to not overload this, and to not have you fall into the rabbit hole of our buildsystem.

Reply:
Thanks for explanation. I was just curious how it is gonna fail when I remove this. And it seems it worked without it just fine :)